src/main/java/lotto/domain/Rank.java

```diff
@@ -17,39 +17,43 @@
     private final Money prize;
 
 
-    Rank(int matchCount, int prize) {
+    Rank(final int matchCount, final int prize) {
         this.matchCount = matchCount;
         this.prize = new Money(prize);
     }
 
-    public static Rank countMatch(Winning winning, Lotto lotto) {
-        int lottoCount = (int) winning.winningLottoNumber(lotto);
-        boolean bonusBallMatch = winning.bonusLottoNumber(lotto);
+    public static Rank countMatch(final Winning winning, final Lotto lotto) {
+        int lottoCount = (int) winning.winLottoNumber(lotto);
+        boolean bonusBallMatch = winning.isbonusLottoNumber(lotto);
         if (THIRD.match(lottoCount) && !bonusBallMatch) {
             return THIRD;
         }
         return Arrays.stream(Rank.values()).filter(rank -> rank.match(lottoCount)).findFirst()
             .orElse(NOTHING);
     }
 
-    public static boolean get(Rank rank) {
-        return rank != rank.NOTHING;
+    public static boolean get(final Rank rank) {
+        return rank != NOTHING;
     }
 
-    public boolean match(int count) {
+    public boolean match(final int count) {
         return matchCount == count;
     }
 
     public int getMatchCount() {
         return matchCount;
     }
 
-    public Money getPrize() {
-        return prize;
+    public int getPrize() {
+        return prize.getValue();
     }
 
-    public Money getPrize(int hit) {
-        return new Money(prize.getValue() * hit);
+    public int getPrize(final int hit) {
+        return new Money(prize.getValue() * hit).getValue();
```

Comment:
`int`를 바로 반환하는데 `Money` 객체로 한번 감싸주고 `getValue()`를 해줄 필요가 있을까요?
```suggestion
        return prize.getValue() * hit;
```